docs/development/contributing_guide.md

```diff
@@ -95,9 +72,9 @@ Fix your favorite problem or perhaps find a [Good First Issue](https://github.co
 to work on.
 
 
-# 7. Turn coffee and documentation into code and documentation!
+# 7. Turn coffee into code and documentation!
 
-There is a growing amount of documentation located in the [`docs`](docs)
+There is a growing amount of documentation located in the `docs`
```

Comment:
There's quite a few links below that also no longer work from the rendered documentation. I think it might be best to translate them to github links to the `develop` branch rather than just inline code block - to get people looking at the repo early?